.travis.yml

```diff
@@ -24,15 +56,39 @@ jobs:
     - MB_PYTHON_VERSION=3.7
     - MB_ML_VER=2014
     - PLAT=aarch64
-  - env:
+    - DOCKER_TEST_IMAGE=multibuild/xenial_arm64v8
+  - os: linux
+    env:
     - MB_PYTHON_VERSION=3.8
+  - os: linux
+    env:
+    - MB_PYTHON_VERSION=3.8
+    - PLAT=i686
   - arch: arm64
     os: linux
     services: docker
     env:
     - MB_PYTHON_VERSION=3.8
     - MB_ML_VER=2014
     - PLAT=aarch64
+    - DOCKER_TEST_IMAGE=multibuild/xenial_arm64v8
+  - os: osx
+    osx_image: xcode11
+    language: generic
+    env:
+    - MB_PYTHON_VERSION=2.7
+  - os: osx
+    osx_image: xcode11
+    language: generic
+    env:
+    - MB_PYTHON_OSX_VERSION=10.6
+    - PLAT=x86_64
```

Comment:
Explicitly required after the multibuild upgrade, but only for Python 3.5.  Wheel building on this configuration was failing due to an inability on the part of clang to build binaries.  Wound up leveraging the "claim 32/64 bit support, only implement 64-bit" option supported by multibuild, so while this wheel claims to support both it really only works on 64-bit platforms.  Hopefully the intersection of Python 3.5 users + OS X + 32-bit is fairly small.